### DISCUSSION TITLE
Sensor for target charge level?

### DISCUSSION THREAD

**Discussion by hogsterbogster** [tomasmcguinness/homeassistant-mixergy#37] [Ideas]
Hi! 😊

It would be great if it's possible to access the 'target charge level' value and pull this into HA – at the moment the only way I can see whether my heat command has been successful is to look at the 'Electric Heat' or 'Mixergy Electric Heat Power' sensors, but neither of these confirm that the tank has received the correct target request.

Many thanks!

David

**Comment by tomasmcguinness** [comment]
Defiantly on the list!
